project-canvas/src/components/backlog-view/Item.tsx

```diff
@@ -1,32 +1,128 @@
-import { Box, Text } from "@mantine/core"
 import { Issue } from "project-extender"
+import { Box, Stack, Text, ThemeIcon } from "@mantine/core"
+import { IconBookmark, IconCheck, IconBug } from "@tabler/icons"
 import { Draggable } from "react-beautiful-dnd"
 
-export function Item({ issueKey, summary, index, status, columnId }: Issue) {
+export function Item({
+  issueKey,
+  summary,
+  index,
+  status,
+  type,
+  storyPoints,
+}: Issue) {
+  let icon: JSX.Element
+  let iconGradient1: string
+  let iconGradient2: string
+  switch (type) {
+    case "Story":
+      icon = <IconBookmark />
+      iconGradient1 = "teal"
+      iconGradient2 = "lime"
+      break
+    case "Task":
+      icon = <IconCheck />
+      iconGradient1 = "teal"
+      iconGradient2 = "blue"
+      break
+    case "Bug":
+      icon = <IconBug />
+      iconGradient1 = "orange"
+      iconGradient2 = "red"
+      break
+
+    default:
+  }
+
   return (
     <Draggable draggableId={issueKey} index={index}>
       {(provided) => (
         <Box
           sx={{
-            backgroundColor: "#eee",
-            borderRadius: 4,
-            padding: "4px 8px",
+            backgroundColor: "white",
+            borderRadius: 2,
+            padding: "5px",
             transition: "background-color .8s ease-out",
-            marginTop: 8,
+            margin: 8,
+            boxShadow:
+              "0 0 1px 0 rgb(9 30 66 / 31%), 0 2px 4px -1px rgb(9 30 66 / 25%)",
 
             ":hover": {
-              backgroundColor: "#fff",
+              backgroundColor: "#ebecf0",
               transition: "background-color .1s ease-in",
             },
           }}
           ref={provided.innerRef}
           {...provided.draggableProps}
           {...provided.dragHandleProps}
         >
-          <Text>{summary}</Text>
-          <Text>{status}</Text>
-          <Text>{issueKey}</Text>
-          <Text>{columnId}</Text>
+          <Stack sx={{ gap: "0px" }}>
+            <Box
+              sx={{
+                display: "flex",
+                alignItems: "center",
+                height: "22px",
+                margin: "5px",
+              }}
+            >
+              <ThemeIcon
+                size="lg"
+                variant="gradient"
+                gradient={{ from: iconGradient1, to: iconGradient2, deg: 105 }}
+                sx={{
+                  width: "16px",
+                  height: "16px",
+                  minHeight: "16px",
+                  minWidth: "16px",
```

Comment:
here `size` was set to `lg` which according to Mantine is `Predefined width and height or number for width and height in px` but then was overwritten with `width` and `height`. This is redundant. Keep the use of `sx` to a strict minimum. No need to set the minHeight and minWidth too.

Reply:
we need minHeight and min Widht because when resizing then right / left Pannel the user would not be able to read the Content anymore